addon/helpers.js

```diff
@@ -0,0 +1 @@
+../chrome/content/helpers.js
```

Comment:
Please remove this file.  Isn't needed anymore now that we're symlinking to the content dir.